### PR TITLE
Fix #4771 - webui callback process error

### LINKED ISSUE #4771: Failed doing webui callback - TypeError: expected string or buffer
**Branch/Commit**: master / 93923792e8908427344d6e01de968656cf096626
**OS**: Ubuntu 14.04
**Browser**: Chrome
**What you did**: Process item from sabnzbd to SickRage (automatically).  It's done this on 3 separate shows.
**What happened**: When I click on the entry in sabnzbd, it shows me this
```
Loading config from /home/spaz/.sickrage/autoProcessTV/autoProcessTV.cfg

Opening URL: https://192.168.1.225:8081/home/postprocess/processEpisode?nzbName=Jeopardy.2018.06.12.720p.HDTV.x264-NTb.nzb&quiet=1&dir=%2Fhome%2Fspaz%2FDownloads%2Ftv%2FJeopardy+2018+06+12+720p+HDTV+x264-NTb
Unable to open URL: HTTP Error 500: Internal Server Error
```
SickRage logs indicate what's pasted below.
**What you expected**: Shows to be sorted properly.  This was working as of 2 days ago, with zero changes made on my part.

**Logs**:
```
2018-06-13 18:25:34 ThreadPoolExecutor-0_0 :: [9392379] Failed doing webui callback: Traceback (most recent call last):
  File "/home/spaz/.sickrage/sickbeard/webserve.py", line 288, in async_call
    result = function(**kwargs)
  File "/home/spaz/.sickrage/sickbeard/webserve.py", line 2483, in processEpisode
    process_path = ss(kwargs.get('dir', xhtml_unescape(proc_dir)) or '')
  File "/home/spaz/.sickrage/lib/tornado/escape.py", line 68, in xhtml_unescape
    return re.sub(r"&(#?)(\w+?);", _convert_entity, _unicode(value))
  File "/usr/lib/python2.7/re.py", line 151, in sub
    return _compile(pattern, flags).sub(repl, string, count)
TypeError: expected string or buffer

2018-06-13 18:13:09 ThreadPoolExecutor-0_3 :: [9392379] Failed doing webui callback: Traceback (most recent call last):
  File "/home/spaz/.sickrage/sickbeard/webserve.py", line 288, in async_call
    result = function(**kwargs)
  File "/home/spaz/.sickrage/sickbeard/webserve.py", line 2483, in processEpisode
    process_path = ss(kwargs.get('dir', xhtml_unescape(proc_dir)) or '')
  File "/home/spaz/.sickrage/lib/tornado/escape.py", line 68, in xhtml_unescape
    return re.sub(r"&(#?)(\w+?);", _convert_entity, _unicode(value))
  File "/usr/lib/python2.7/re.py", line 151, in sub
    return _compile(pattern, flags).sub(repl, string, count)
TypeError: expected string or buffer
```

### PR DESCRIPTION
**Pull request by Fiat2LTC**
Fixes #4771

Proposed changes in this pull request:
- Fix #4771 - webui callback process error

- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read [contribution guide](https://github.com/SickRage/SickRage/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

**Comment by MilhouseVH** [comment]
No exception now, thanks!

**Comment by TheHolyRoger** [comment]
@miigotu any chance of a merge please?

**Comment by ctrlbrk42** [comment]
@miigotu my sickrage has been broken for importing sabnzbd downloads for a month since the last build, can you please merge this?

File: sickbeard/webserve.py [hunk 1]
```diff
@@ -2480,7 +2480,7 @@ def processEpisode(self, proc_dir=None, nzbName=None, quiet=None, process_method
                        is_priority=None, delete_on="0", failed="0", proc_type="manual", force_next=False, *args_, **kwargs):
 
         mode = kwargs.get('type', proc_type)
-        process_path = ss(kwargs.get('dir', xhtml_unescape(proc_dir)) or '')
+        process_path = xhtml_unescape(ss(kwargs.get('dir', proc_dir) or ''))
```
Comment (miigotu) [inline sickbeard/webserve.py]: I think this should be inside the ss()

    process_path = ss(xhtml_unescape(kwargs.get('dir', proc_dir) or ''))

**Comment by Fiat2LTC** [comment]
Done @miigotu !

**Pull request merged**